lib/HAPBridgeNode.js

```diff
@@ -156,7 +156,7 @@ module.exports = function(RED) {
 }
 
 function macify(nodeId) {
-    const noDecimalStr = nodeId.replace('.', '')
-    const paddedStr = noDecimalStr.padStart(16, '0')
-    return paddedStr.match(/.{1,2}/g).join(':')
+    const noDecimalStr = nodeId.replace('.', '');
+    const paddedStr = noDecimalStr.padEnd(12, '0");
```

Comment:
Does it work? I can see single quote and double quote used.

Reply:
A big oops yesterday, I was trying to be quick with the browser editor, rather than using the proper tools